include/iotconnect_event.h

```diff
@@ -15,16 +15,22 @@ extern "C" {
 #endif
 
 typedef enum {
-    UNKNOWN_EVENT = 0,
-    DEVICE_COMMAND = 0x01,
-    DEVICE_OTA = 0x02,
-    MODULE_UPDATE_COMMAND = 0x03,
-    ON_CHANGE_ATTRIBUTE = 0x10,
-    ON_CHANGE_SETTING = 0x11,
-    ON_FORCE_SYNC = 0x12,
-    ON_ADD_REMOVE_DEVICE = 0x13,
-    ON_ADD_REMOVE_RULE = 0x15,
-    ON_CLOSE = 0x99
+    DEVICE_COMMAND = 0,
+    DEVICE_OTA = 1,
+    MODULE_UPDATE_COMMAND = 2,
+    ON_CHANGE_ATTRIBUTE = 101,
+    REFRESH_SETTING_TWIN = 102,
+    RULE_CHANGE_COMMAND = 103,
+    REFRESH_CHILD_DEVICE = 104,
+    DF_CHANGE = 105,
+    DEVICE_DELETE = 106,
+    DEVICE_DISABLE = 107,
+    ON_CLOSE = 108,
+    STOP_OPERATION = 109,
+    HEARTBEAT_COMMAND = 110,
+    STOP_HEARTBEAT = 111,
+    UNKNOWN_EVENT = 0x10,
+    ON_FORCE_SYNC = 0x12
```

Comment:
Not a massive deal, but I think this enum definition could use some tidying e.g. keep everything in the same radix (decimal or hexadecimal), UNKNOWN_EVENT and ON_FORCE_SYNC should be defined underneath MOBILE_UPDATE_COMMAND to keep things in order.